cmd/main.go

```diff
@@ -121,8 +115,9 @@ func prepareWorkPathAndCustomConf(action cli.ActionFunc) func(ctx *cli.Context)
 func NewMainApp(version, versionExtra string) *cli.App {
 	app := cli.NewApp()
 	app.Name = "Gitea"
+	app.HelpName = "gitea"
 	app.Usage = "A painless self-hosted Git service"
-	app.Description = `By default, Gitea will start serving using the web-server with no argument, which can alternatively be run by running the subcommand "web".`
+	app.Description = `Gitea program contains "web" and other subcommands. If no subcommand is given, it starts the web server by default. Use "web" subcommand for more web server arguments, user other subcommands for other features.`
```

Comment:
1.
```suggestion
	app.Description = `The gitea binary contains multiple subcommands, among them "web" . If no subcommand is given, the web server is started. Use the "web" subcommand to configure how the web server is started, or another subcommand for other purposes.`
```
2. Didn't we remove the functionality of the second sentence?

Reply:
I am not a fan of using "the" everywhere.

See the screenshot and old sentences, we never say "the Gitea web server", "Dump the Gitea files", "output the CLI document".


> When to Use 'A,' 'An,' or 'The'

https://gallaudet.edu/student-success/tutorial-center/english-center/grammar-and-vocabulary/definite-and-indefinite-articles/when-to-use-a-an-or-the/

Reply:
The reason that I think we should put "web" subcommand before others: it is the default one and is the only one most users really need.